examples/simple-moco-project/mocks/moco.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "response": {
+            "text": "Hi, I'm the moco container :D"
```

Comment:
What do you think about `Hi, I'm a unicorn, BRO.`? 😄

Reply:
I love this idea Bro, I'll do it soon !!! :)